KabCollections/src/main/java/application/rest/CollectionsAccess.java

```diff
@@ -124,7 +125,29 @@ public Response listCollections(@Context final HttpServletRequest request) {
 		return Response.ok(msg).build();
 	}
 
-
+	@POST
+	@Produces(MediaType.APPLICATION_JSON)
+	@Consumes(MediaType.APPLICATION_JSON)
+	@Path("/onboard")
+	public Response onboardDeveloper(@Context final HttpServletRequest request,
+			                         final JSONObject jsonInput) {
+		System.out.println("if only we could onboard you...");
+		String gituser = (String) jsonInput.get("gituser");
+		System.out.println("gituser: \"" + gituser + "\"");
+		String gittoken = (String) jsonInput.get("gittoken");
+		System.out.println("gittoken: \"" + gittoken + "\"");
+		String workaround = "Command development in progress, please go to the tekton dashboard in your browser and manually configure the webhook";
```

Comment:
Onboard also requires repoName of the repo Champ will onboard for Jane

Reply:
Git token is not necessary according to Alan for onboarding, just reponame and git ID